crypto/x509/x_all.c

```diff
@@ -138,12 +175,60 @@ X509 *X509_load_http(const char *url, BIO *bio, BIO *rbio, int timeout)
                                    ASN1_ITEM_rptr(X509));
 }
 
+/*
+ * Drop empty SKID or AKID X509_REQ extensions that might result from:
+ * "...KeyIdentifier = none"
+ */
+
+static void drop_empty_req_keyid_exts(X509_REQ *x)
```

Comment:
I see this function for PKCS#10 CSR pretty irrelevant because
placing SKIDs in CSRs makes little sense to me anyway, since usually the cert producer takes care of this.
Placing AKIDs in CSRs makes no sense at all because the cert signing key is not known/available there.

Reply:
You meanwhile removed `drop_empty_req_keyid_exts()`,
but actually my above comment also holds for the new calls to `bad_keyid_exts(exts)` in `X509_REQ_sign{,_ctx}()`: handling any SKID and AKID in CSRs has little/no relevance.